pkg/util/procfs/procfs_test.go

```diff
@@ -66,7 +69,27 @@ func TestPidOf(t *testing.T) {
 	if runtime.GOOS == "darwin" || runtime.GOOS == "windows" {
 		t.Skipf("not supported on GOOS=%s", runtime.GOOS)
 	}
-	pids := PidOf(filepath.Base(os.Args[0]))
+	pids, err := PidOf(filepath.Base(os.Args[0]))
+	assert.Empty(t, err)
 	assert.NotZero(t, pids)
 	assert.Contains(t, pids, os.Getpid())
 }
+
+func TestPKill(t *testing.T) {
+	if runtime.GOOS == "darwin" || runtime.GOOS == "windows" {
+		t.Skipf("not supported on GOOS=%s", runtime.GOOS)
+	}
+	sig := syscall.SIGCONT
+	c := make(chan os.Signal, 1)
+	signal.Notify(c, sig)
+	defer signal.Stop(c)
+	PKill(os.Args[0], sig)
+	select {
+	case s := <-c:
+		if s != sig {
+			t.Fatalf("signal was %v, want %v", s, sig)
+		}
+	case <-time.After(1 * time.Second):
```

Comment:
@yujuhong : please see https://github.com/golang/go/blob/master/src/os/signal/signal_test.go#L27 and the TestSIGCONT from where waitSig is called. If you still want me to change it, please let me know.